test/e2e/pkg/infra/docker/docker.go

```diff
@@ -47,6 +47,32 @@ func (p Provider) StopTestnet(ctx context.Context) error {
 	return ExecCompose(ctx, p.Testnet.Dir, "down")
 }
 
+// FIXME: do we need this method in v0.38?
+func (p Provider) Disconnect(ctx context.Context, name string, _ string) error {
```

Comment:
This method is not used in this branch, apparently.

So we could remove it from this PR?

The same is valid for the following 3 FIXMEs.